libraries/AP_HAL_ChibiOS/UARTDriver.cpp

```diff
@@ -478,6 +479,47 @@ void UARTDriver::_begin(uint32_t b, uint16_t rxS, uint16_t txS)
     }
 }
 
+void UARTDriver::_begin_framing(uint16_t _frame_size, uint16_t frames_in_buffer)
+{
+    _frame_readbuf.clear();
+    _frame_readbuf.set_size((_frame_size + sizeof(uint32_t)) * frames_in_buffer);
```

Comment:
why the +4?

Reply:
It's the length of the frame. frames can be variable length remember - the size is just the maximum size

Reply:
Removed